Podcast/ProfileActivityTableViewCell.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ProfileActivityTableViewCell.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/10/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class ProfileActivityTableViewCell: UITableViewCell {
+    
+    let height: CGFloat = 70
+    let contentPaddingX:CGFloat = 6.0
+    let contentPaddingY:CGFloat = 3.0
+    
+    var actLabel: UILabel!
+    var dateLabel: UILabel!
+    var containerView: UIView!
+    var podImage: UIImageView!
+    
+    var activity: UserActivity? {
+        didSet {
+            if let activity = activity {
+                let dateFormatter = DateFormatter()
+                dateFormatter.dateFormat = "eee, MMM dd"
+//                dateFormatter.timeStyle = .none
+                
+                dateFormatter.locale = Locale(identifier: "en_US")
+                dateLabel.text = dateFormatter.string(from: activity.date as Date)
+//                print(dateFormatter.string(from: activity.date as Date))
+                
+                // Check for image
+                podImage.image = activity.episode.smallArtworkImage
+                
+//                let podcastName = NSAttributedString(string: activity.episode.title, attributes: [NSForegroundColorAttributeName: UIColor.podcastPinkText, NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)])
+//                let seriesName = NSAttributedString(string: (activity.episode.series?.publisher?.name)!, attributes: [NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)])
+//                let authorName = NSAttributedString(string: activity.episode.title, attributes: [NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)])
+                let podcastName = activity.episode.title
+//                let seriesName = (activity.episode.series?.title)!
+//                let authorName = (activity.episode.series?.publisher?.name)!
+                var actText = NSMutableAttributedString()
+                switch (activity.action) {
+                case .Liked:
+                    actText = NSMutableAttributedString(string: "Liked "+podcastName)
```

Comment:
I would store the string `"Liked"` in a separate variable and use it to get the location `6` for the `NSRange` in line 46, since it's subject to change